frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -474,77 +474,75 @@ export function SurveyResult({ disableEventsTable }: { disableEventsTable?: bool
     } = useValues(surveyLogic)
 
     return (
-        <>
-            <>
-                <Summary surveyUserStatsLoading={surveyUserStatsLoading} surveyUserStats={surveyUserStats} />
-                {survey.questions.map((question, i) => {
-                    if (question.type === SurveyQuestionType.Rating) {
-                        return (
-                            <>
-                                {question.scale === 10 && (
-                                    <>
-                                        <div className="text-4xl font-bold">{surveyNPSScore}</div>
-                                        <div className="mb-2 font-semibold text-secondary">Latest NPS Score</div>
-                                        <SurveyNPSResults survey={survey as Survey} />
-                                    </>
-                                )}
-
-                                <RatingQuestionBarChart
-                                    key={`survey-q-${i}`}
-                                    surveyRatingResults={surveyRatingResults}
-                                    surveyRatingResultsReady={surveyRatingResultsReady}
-                                    questionIndex={i}
-                                    iteration={survey.current_iteration}
-                                />
+        <div className="space-y-4">
+            <Summary surveyUserStatsLoading={surveyUserStatsLoading} surveyUserStats={surveyUserStats} />
+            {survey.questions.map((question, i) => {
+                if (question.type === SurveyQuestionType.Rating) {
+                    return (
+                        <>
+                            {question.scale === 10 && (
```

Comment:
high number of line changes is misleading here. I removed a fragment, so nesting shifted left, but the only change here is adding `space-y-4` to the parent div

Reply:
cool, the former does `mb-4` or `mb-8`, but now it is all 4. Are we breaking/changing the UI unintentionally?

Reply:
I changed it intentionally to make it consistent. IMO it was a little bit weird to have some of the charts separated by mb-8 and others with mb-4.

Reply:
> no functionality nor UI changes, it's just a code refactor

cool, let's be clearer on the PR. desc. then since the above is the opposite of what the PR does.